examples/mcp-client/strands/main.py

```diff
@@ -0,0 +1,65 @@
+"""
+AWS IAM MCP Client Example: Strands Agent Integration
```

Comment:
Can you remove the name `AWS IAM MCP` from the PR and make it more generic. We don't own this name from a trademark perspective. I did a pass but will have missed some :)